src/main/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtils.java

```diff
@@ -72,13 +75,13 @@ public static boolean isPredefinedShelf(String shelfName) {
     /**
      * Fetches all of the books in the chosen predefined shelf
      */
-    public Set<Book> getBooksInChosenPredefinedShelf(String chosenShelf) {
+    public static Set<Book> getBooksInChosenPredefinedShelf(String chosenShelf) {
         Set<Book> books;
         if (isAllBooksShelf(chosenShelf)) {
             return getBooksInAllPredefinedShelves();
         }
 
-        PredefinedShelf.ShelfName predefinedShelfName = getPredefinedShelfName(chosenShelf);
+        PredefinedShelf.ShelfName predefinedShelfName = PredefinedShelfUtils.getPredefinedShelfName(chosenShelf);
```

Comment:
Since we are calling the method with in same class we don't need to use class name?